src/main.js

```diff
@@ -92,12 +96,31 @@ export class Draggable {
             const { which } = e;
 
             if ((which && which > 1) || this._ignoreMouse) {
+                bind(this.document, "contextmenu", preventDefault);
+                
+                this.cancelDrag(e);
+
+                this._cancelHandler(e);
+
                 return;
             }
 
-            bind(this.document, "mousemove", this._mousemove);
-            bind(this.document, "mouseup", this._mouseup);
-            this._pressHandler(e);
+            if (!this._clickMoveClick) {
+                bind(this.document, "mouseup", this._mouseup);
+                bind(this.document, "mousemove", this._mousemove);
+                bind(this.document, "contextmenu", preventDefault);
```

Comment:
This event wasn't prevented in the original version (and will prevent the "contextmenu", which might be interpreted as a regression) - it seems better not to have it here, but only when the `lick-move-click` dragging option is enabled. Am I missing something?